### DISCUSSION TITLE
Ask for solutions:ModuleNotFoundError: No module named 'openai'

### DISCUSSION THREAD

**Discussion by Bob-wentao** [openai/openai-python#796] [Q&A]
![image](https://github.com/openai/openai-python/assets/143519535/1aa0cec5-a8d7-4679-b082-7b724faa3b73)
I have installed pip3 and using "pip3 install openai" in the terminal.
However ,it still doesn't work.
Could you be so kinds to help me!

**Comment by rattrayalex** [comment]
I suggest the OpenAI Discord for help getting your Python environment set up!
